pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -82,6 +102,8 @@ func (ds *dockerService) RunPodSandbox(config *runtimeapi.PodSandboxConfig) (str
 		return "", fmt.Errorf("failed to create a sandbox for pod %q: %v", config.Metadata.Name, err)
 	}
 
+	ds.setNetworkReady(createResp.ID, false)
```

Comment:
I cannot think of a better way to address this. Not much options:

1. kubelet needs to know the RunPodSandbox has not finish. Do not call PodSandboxStatus (Not possible since sandbox ID is unknown to kubelet)
2. dockershim needs to know RunPodSandbox has not finish. Skip getting ip from network plugin. 

I am willing to settle with the current solution.